am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/DefaultPermissionGrant.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import javafx.util.Pair;
```

Comment:
Please use different class. Implement your own when needed. We should not relay on JavaFX

Reply:
Project does not even compile on newer JDK. JavaFX got removed from JDK in later versions.